web/src/components/SurveyListTable.tsx

```diff
@@ -0,0 +1,181 @@
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+`;
+
+const HeadItem = styled.th`
+  margin: 2px;
+  padding: 18px;
+  font-size: 17px;
+  font-weight: bold;
+  text-align: center;
+  color: ${(props) => props.theme.colors.default};
+`;
+
+const Title = styled(Item)`
+  min-width: 15vh;
+  flex: 1;
+  cursor: pointer;
+
+  &:hover {
+    background-color: ${(props) => props.theme.colors.btnhover};
+  }
+`;
+
+const AuthList = styled(Item)`
+  min-width: 100px;
+  width: 20vw;
+`;
+
+const EndDate = styled(Item)`
+  min-width: 150px;
+  width: 13vw;
+  text-align: center;
+
+  @media screen and (max-width: 900px) {
+    display: none;
+  }
+`;
+
+const HeadTitle = styled(HeadItem)`
+  min-width: 15vh;
+  flex: 1;
+`;
+
+const HeadAuthList = styled(HeadItem)`
+  min-width: 100px;
+  width: 20vw;
+`;
+
+const HeadEndDate = styled(HeadItem)`
+  min-width: 150px;
+  width: 13vw;
+
+  @media screen and (max-width: 900px) {
+    display: none;
+  }
+`;
+
+const Notification = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  margin-top: 35vh;
+  padding: 10px;
+`;
+
+const Label = styled.label`
+  font-size: 50px;
+  font-weight: 700;
+  color: ${(props) => props.theme.colors.default};
+  text-align: center;
+
+  @media screen and (max-width: 700px) {
+    font-size: 30px;
+  }
+`;
+
+interface SurveyListTableProps {
+  surveys: SurveyAbstractResponse[];
+  setSelectedSurveyIndex: (arg: number) => void;
+  setPreviewModalOpen: (arg: boolean) => void;
+  theme: DefaultTheme;
+}
+
+export default function SurveyListTable({
+  surveys,
+  setSelectedSurveyIndex,
+  setPreviewModalOpen,
+  theme,
+}: SurveyListTableProps) {
+  const navigate = useNavigate();
+
+  const handleButtonClick = (index: number) => {
+    setSelectedSurveyIndex(index);
+    setPreviewModalOpen(true);
+  };
+
+  if (surveys.length === 0) {
+    return (
+      <Notification theme={theme}>
+        <Label theme={theme}>😥 참여가능한 설문이 없습니다...</Label>
+        <br />
+        <RectangleButton
+          text="설문 만들러 가기"
+          width="250px"
+          backgroundColor={theme.colors.primary}
+          hoverColor={theme.colors.prhover}
+          handleClick={() => navigate('/survey/form')}
+          theme={theme}
+        />
+      </Notification>
+    );
+  }
+
+  return (
+    <ListTable theme={theme}>
+      <ListHead>
+        <ListRow>
+          <HeadTitle theme={theme}>설문 제목</HeadTitle>
+          <HeadAuthList theme={theme}>필수인증</HeadAuthList>
+          <HeadEndDate theme={theme}>설문 종료일</HeadEndDate>
+        </ListRow>
+      </ListHead>
+
+      <ListBody>
+        {surveys.map((survey: SurveyAbstractResponse, index: number) => (
+          <ListRow key={survey.surveyId} theme={theme}>
+            <Title role="button" onClick={() => handleButtonClick(index)} theme={theme}>
+              {survey.title}
+            </Title>
+            <AuthList theme={theme}>
```

Comment:
Could you separate this `<AuthList>` component? This is the same reason for your separated component of `<Pagination>`